packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -113,18 +114,16 @@ const LinkControlSearchInput = forwardRef(
 			}
 		};
 
-		const inputLabel = placeholder ?? __( 'Search or type URL' );
-
 		return (
 			<div className="block-editor-link-control__search-input-container">
 				<URLInput
 					disableSuggestions={ currentLink?.url === value }
-					label={ inputLabel }
+					label={ label ?? __( 'Search or type URL' ) }
 					hideLabelFromVision={ hideLabelFromVision }
 					className={ className }
 					value={ value }
 					onChange={ onInputChange }
-					placeholder={ inputLabel }
```

Comment:
My understanding was that if there's a `label` then the placeholder doesn't need to be present. I feel it should be up to the consumer of the component to choose to add some text here. It should be an _example_ of the type of input the control expects rather than repeating the string `Search` from label.

Reply:
I can make it so that the placeholder field is set to null as a fallback.

Reply:
There are a couple of `playwrite` tests implemented which are using this field's default value as a method of locating the component. What should be the approach here?

Reply:
I would use https://playwright.dev/docs/locators#locate-by-label